modules/ducktests/tests/docker/Dockerfile

```diff
@@ -48,14 +48,13 @@ ARG APACHE_MIRROR="https://apache-mirror.rbc.ru/pub/apache/"
 ARG APACHE_ARCHIVE="https://archive.apache.org/dist/"
 
 # Install binary test dependencies.
-ARG IGNITE_VERSION="2.8.0"
-ARG IGNITE_NAME="ignite-$IGNITE_VERSION"
-ARG IGNITE_RELEASE_NAME="apache-ignite-$IGNITE_VERSION-bin"
-
-ADD $APACHE_ARCHIVE/ignite/$IGNITE_VERSION/$IGNITE_RELEASE_NAME.zip /opt/
-RUN cd /opt && unzip $IGNITE_RELEASE_NAME.zip && rm $IGNITE_RELEASE_NAME.zip
-RUN mv /opt/$IGNITE_RELEASE_NAME /opt/$IGNITE_NAME
-RUN chmod a+wr /opt/$IGNITE_NAME -R
+ADD $APACHE_ARCHIVE/ignite/2.7.6/apache-ignite-2.7.6-bin.zip /opt/
```

Comment:
Use "RUN curl -O" or "RUN wget" instead of ADD to enable caching. Check this [issue](https://github.com/moby/moby/issues/15717)

Reply:
Done